discord/no-decay/index.html

```diff
@@ -756,19 +681,18 @@ <h1 class="subtitle">
                 </div>
             </div>
         </section>
-        <div id="hostBanner">
+        <!-- <div id="hostBanner">
             <figure class="image is-128x128" >
                 <a href="https://github.com/Tensounder54/Tensounder54.github.io">
                     <img src="http://datampi.org/images/hosted-on-github.png" alt="Hosted On GitHub">
                 </a>
             </figure>
-        </div>
+        </div> -->
         <div id="menu">
             <div class="container">
                 <div class="columns is-mobile">
                     <div class="column is-one-half">
                         <ul>
-                            <!-- <li><a href="#" class="Disabled button is-unselectable" disabled>Menu</a></li> -->
```

Comment:
I want to leave this in here in case I want to enable it in the future. Is there a reason that it should be removed?

Reply:
So having lots on unnecessary comments breaks one of the principles of clean code and makes it harder to read. Also you have source control so you can also go back and find this later if you need it :)